ambry-cloud/src/main/java/com.github.ambry.cloud/CloudBlobMetadata.java

```diff
@@ -80,6 +81,24 @@ public CloudBlobMetadata() {
    */
   public CloudBlobMetadata(BlobId blobId, long creationTime, long expirationTime, long size,
       EncryptionOrigin encryptionOrigin, String vcrKmsContext, String cryptoAgentFactory) {
+    this(blobId, creationTime, expirationTime, size, encryptionOrigin, vcrKmsContext, cryptoAgentFactory, -1);
+  }
+
+  /**
```

Comment:
If we're going to have two constructors, let's make the first one be without encryption (defaults/nulls for those params) and the other with all the params.

Reply:
fixed.